src/home/fetch-github/fetch-and-display-previews.ts

```diff
@@ -48,6 +48,7 @@ export async function fetchAvatars() {
   const urlPattern = /https:\/\/github\.com\/(?<org>[^/]+)\/(?<repo>[^/]+)\/issues\/(?<issue_number>\d+)/;
 
   const avatarPromises = cachedTasks.map(async (task) => {
+    if (!task.preview.body) return;
```

Comment:
There's a lot of changes in here that are not wanted according to the spec. Be sure to remove things like this and the toggle button, and css styles related to the private label etc. 

All you should do is seamlessly display the private repo tasks for now. 

Screen real estate is at a huge premium for mobile especially. I also don't know if private repos really need to be highlighted per se.

Reply:
Okay, i'd remove the toggle and private tag.. this highlighted code though is there because i switched from using a custom `GithubIssue` interface to the official so it needs to be checked if the body is empty before continuing

Reply:
@pavlovcik I've forced pushed it back to the commit with no toggles or private tags

Reply:
This is still here. Why did you add this?

Reply:
> This is still here. Why did you add this?

`preview.body maybe null`

Reply:
> This is still here. Why did you add this?

It throws error after i switch to thew official `Issue` type

Reply:
Just a vs code error though, i can remove it if it don't look good.. just a check it needs